### ISSUE TITLE
Active/inactive borders don't work properly with yabai autofocus.

### ISSUE THREAD

**Issue by xfiver** [FelixKratz/JankyBorders#40]
The border switches to the active state only after clicking on the window with the inactive border, instead of following active window.
![2023-12-17 01 43](https://github.com/FelixKratz/JankyBorders/assets/38538855/3fa8e42f-edd1-410d-b765-952905e4d30a)

**Comment by FelixKratz** [comment]
Duplicate of #15
